### DISCUSSION TITLE
Problem with caddy start

### DISCUSSION THREAD

**Discussion by hendrikjap** [Budibase/budibase#3833] [Help] [answered]
Hi Guys,

Kindly see the screenshot. I am not sure what I am doing wrong. 

![image](https://user-images.githubusercontent.com/55825618/146643141-7c1bca74-c0b8-43c4-a72f-707b4df6398a.png)

**Comment by dewdeded** [comment, marked as answer]
The solution is in the last line. Caddy can't start, because another service (webserver) is using port 80.
 
Per default Caddy tries to use this http-Port for http-to-https redirecting.

If u you don't want oder need that, you can add:

{
	auto_https disable_redirects
}
in your config and it wont try to bind port 80.

**Reply by hendrikjap** [reply]
Thanks!

**Reply by HideoKuze** [reply]
Will this then stop Caddy from adding TLS to the server? I'm having this issue too, Budibase uses port 80, and I can't use Caddy to add TLS
